lib/Sema/ConstraintSystem.cpp

```diff
@@ -1711,6 +1713,15 @@ ConstraintSystem::getTypeOfReference(ValueDecl *value,
     auto numLabelsToRemove = getNumRemovedArgumentLabels(
         funcDecl, /*isCurriedInstanceReference=*/false, functionRefKind);
 
+    if (ctx.LangOpts.hasFeature(Feature::InferSendableMethods)) {
+      // All global functions should be @Sendable
+      if (!funcDecl->getDeclContext()->isTypeContext()) {
+        funcType =
+            funcType->withExtInfo(funcType->getExtInfo().withConcurrent())
+                ->getAs<AnyFunctionType>();
```

Comment:
I don't think we need `getAs<AnyFunctionType>` because `withExtInfo` would produce `AnyFunctionType *`.

Reply:
Yes you're right , withExtInfo produces AnyFunctionType so I'll remove this